app/routes/events/components/EventAdministrate/Abacard.js

```diff
@@ -79,6 +85,10 @@ class Abacard extends React.Component<*, State> {
             this.input = input;
           }}
         />
+        <div className={styles.counter}>
+          {' '}
```

Comment:
do you need this empty space or did prettier just put it in?

Reply:
Prettier put it in. Should I remove it?

Reply:
yeah, I don't think you need it (it's on its own line anyway right, so a space shouldn't matter)